src/image/utils.ts

```diff
@@ -0,0 +1,67 @@
+import * as tf from '@tensorflow/tfjs-core';
+
+interface ImageProp {
+  height: number;
+  width: number;
+  channel: number;
+
+}
+
+interface Img2TensorArgs extends ImageProp {
+  data: Buffer;
+  newChannel: number;
+}
+
+export function img2array(opt: Img2TensorArgs): tf.Tensor3D {
+
+  let imgArray: Array<Array<Array<number>>> = [];
+
+  for (let ch = 0; ch < opt.newChannel;ch++){
+    let channelArray: Array<Array<number>> = [];
+
+    for (let j = 0; j < opt.width; j++){
+      let innerArray: Array<number> = [];
+
+      for (let k = 0; k < opt.height; k++){
+        let index: number = ((opt.width * k) + j) * opt.channel + ch;
+        innerArray.push(opt.data[index]);
+      }
+      channelArray.push(innerArray);
+    }
+    imgArray.push(channelArray);
+  }
+  return tf.tensor3d(imgArray);
+}
```

Comment:
Just made a small benchmark for this piece of code. I found if the data is flattened we should directly use `tf.tensor(data, [W, H, C])` instead of iterating (this is time consuming) the buffer and reconstruct it in 3-D array.
A small piece of testing code:

```js
function m() {
  let t = Date.now();
  for (let i = 0; i < 224 * 224 * 3; i++) {
   data.push(i % 255); 
  }
  tf.tensor(data, [224, 224, 3]);
  console.log(Date.now() - t);

  t = Date.now();
  for (let i = 0; i < 3; i++) {
    let c = []
    for (let j = 0; j < 224; j++) {
      let k = []
      for (let m = 0; m < 224; m++) {
        k.push(m)
      }
      c.push(k)
    }
    data2.push(c)
  }
  tf.tensor3d(data2);
  console.log(Date.now() - t);
}
```